src/compiler/types.ts

```diff
@@ -4304,7 +4306,7 @@ namespace ts {
         SpreadIncludes = Read | Spread,
 
         FirstEmitHelper = Extends,
-        LastEmitHelper = ExportStar
+        LastEmitHelper = ExternalEmitHelpers[" LastPlusOne"] - 1

```

Comment:
You could also just move this next to the last one to make it obvious that these should be updated together.

Reply:
Agreed. 👎 for the magic string in the enum.